R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -721,6 +721,16 @@ test_that(
   expect_true(dropTempView("dfView"))
 })
 
+test_that(
+  "createGlobalTempView() results in a queryable table and sql() results in a new DataFrame", {
+  df <- read.json(jsonPath)
+  createGlobalTempView(df, "global_table1")
+  newdf <- sql("SELECT * FROM global_temp.global_table1 where name = 'Michael'")
+  expect_is(newdf, "SparkDataFrame")
+  expect_equal(count(newdf), 1)
+  expect_true(dropGlobalTempView("global_table1"))
```

Comment:
Ur, if you don't mind, I will use the way to count tables like 
https://github.com/apache/spark/commit/b952b44af4d243f1e3ad88bccf4af7d04df3fc81 .